ci/scripts/run_pytest.sh

```diff
@@ -36,13 +36,6 @@ EOF
     exit "${1:-0}"
 }
 
-# Function to handle cleanup on script exit
-cleanup() {
-    local exit_code=$?
-    [ -n "${coverage_files[*]:-}" ] && rm -f "${coverage_files[@]:-}"
-    exit "$exit_code"
-}
-
```

Comment:
We need these coverage files to persist to the next step in the GHA actions pipeline so we can upload them to codecov. Do these stick around on the blossom workers?